tests/vse_sync_pp/analyzers/test_analyzer.py

```diff
@@ -171,9 +171,10 @@ def method(self):
     def make_test_result(constructor, fqname, expect):
         """Make a function testing analyzer test result and analysis"""
         # pylint: disable=too-many-arguments
-        @params(*expect)
-        def method(self, dct):
+        @params(*enumerate(expect))
+        def method(self, idx, dct):
             """Test analyzer test result and analysis"""
+            msg = f'(item {idx} in `expect`)'
```

Comment:
I suggest adding the `fqname` to `msg`. It's already available in the scope and puts all the important information in a single message:

```suggestion
msg = f'(item {idx} in `expect` of {fqname})'
```